DOCUMENTATION.md

```diff
@@ -0,0 +1,296 @@
+
+For significant changes, it’s often good to discuss in an issue first to ensure the change aligns with the project goals. We aim to keep the library lightweight and focused on CDP Logger interactions.
+
+Also, make sure any contributions you submit are your own work and that you’re okay with releasing them under the project’s MIT license.
+
+## License
+
+This project is open-source software licensed under the **MIT License**. See the [LICENSE](LICENSE) file for details. This means you are free to use, modify, and distribute this library in your own projects, even commercial ones, provided you include the copyright notice and license text in any redistribution. 
+
+## Resources and Helpful Links
+
+- **CDP Studio Documentation – CDP Logger:** To understand the context and capabilities of the CDP Logger component, see the official CDP Studio documentation. The CDP Logger is described as a component for logging values and events for long-term storage. You can read more in the CDP Studio Manual under *Framework - Data Logging*. (Visit the [CDP Studio Docs website](https://cdpstudio.com/manual/cdp/cdplogger/cdplogger-index.html).
+
+- **CDP Studio Official Website:** For more information about CDP Studio in general (the development environment, features, downloads, etc.), visit the [official CDP Studio website](https://cdpstudio.com). This site contains tutorials, user manuals, and other resources to help you get started with building applications in CDP Studio.
+
+- **Example Project:** For a complete example of how to use the CDP Logger Client in a Web environment Vue.js project, see the [Adding a Vue.js Web GUI to Automation System](https://cdpstudio.com/manual/cdp/examples/webui-demo.html).
+
+- **GitHub Repository (Source Code):** You can find the source code for this library, report issues, and see example code on the project’s GitHub page: [CDPTechnologies/JavascriptCDPLoggerClient](https://github.com/CDPTechnologies/JavascriptCDPLoggerClient). Feel free to star the repo and watch for updates. All development and issue tracking is done through GitHub.
+
+- **Contact (Support):** If you have questions or need help with the library, you can reach out to the maintainers via the CDP Technologies support email: **support@cdptech.com**. We encourage you to use the issue tracker on GitHub for bug reports and feature requests, but for direct support inquiries, email is available.
+
+---
+
+*Thank you for using the JavaScript CDP Logger Client!* We hope this library makes it easier to integrate CDP Studio’s powerful logging capabilities into your own tools and applications. ([GitHub - CDPTechnologies/JavascriptCDPLoggerClient: A simple Javascript interface to communicate with CDP applications containing a CDPLogger component to retrieve historic data.](https://github.com/CDPTechnologies/JavascriptCDPLoggerClient#:~:text=A%20simple%20JavaScript%20interface%20for,com))
```

Comment:
The link text is weirdly long - maybe not needed at all here?

![image](https://github.com/user-attachments/assets/6e330639-524b-4113-bcfd-5a39645c3e06)